schemas/common/telemetry/tracing/v1.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$id": "https://mapcolonies.com/common/telemetry/tracing/v1",
+  "title": "commonTelemetryTracingV1",
+  "description": "Telemetry tracing configuration",
+  "type": "object",
+  "properties": {
+    "isEnabled": {
+      "description": "Whether tracing is enabled",
+      "type": "boolean",
+      "x-env-value": "TELEMETRY_TRACING_ENABLED",
+      "default": false
+    },
+    "url": {
+      "description": "The URL of the otlp endpoint",
+      "type": "string",
+      "pattern": "^http(s)?://.*$",
+      "format": "uri",
+      "x-env-value": "TELEMETRY_TRACING_URL"
+    },
+    "traceRatio": {
+      "description": "The ratio of traces to sample",
+      "type": "number",
+      "minimum": 0,
+      "maximum": 1,
+      "x-env-value": "TELEMETRY_TRACING_RATIO"
+    },
+    "debug": {
```

Comment:
Add default value as false for debug

Reply:
there is no need as in this case undefined is also false.

Reply:
OK